bin/node/runtime/src/lib.rs

```diff
@@ -350,7 +350,7 @@ impl pallet_balances::Trait for Runtime {
 	type Event = Event;
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = frame_system::Module<Runtime>;
-	type WeightInfo = weights::pallet_balances::WeightInfo<Runtime>;
+	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
```

Comment:
I liked having substrate node having its own weights, as demonstration for how other on how substrate should be used.
But I understand the current choice.

Reply:
The main reason to do this choice is for automation of generating the weights for this project. Otherwise we would need to generate two files each time, with two different templates, and this does not seem normal.